src/main/java/com/github/manolo8/darkbot/backpage/AuctionManager.java

```diff
@@ -31,11 +34,26 @@ public boolean update(int expiryTime) {
     public boolean update(long expiryTime) {
         try {
             if (System.currentTimeMillis() <= lastAuctionUpdate + expiryTime) return false;
-            String page = backpageManager.getConnection("indexInternal.es?action=internalAuction", Method.GET).getContent();
+            StringJoiner dataHtml = new StringJoiner("\n");
 
-            if (page == null || page.isEmpty()) return false;
+            JsonObject hourObj = backpageManager.getConnection("ajax/auction.php", Method.POST)
+                    .setParam("command", "getAuctionList")
+                    .setParam("category", "hour")
+                    .setParam("list", "auction")
+                    .fromJson(JsonObject.class);
+            if (hourObj.get("result").getAsString().equalsIgnoreCase("ok")) {
+                dataHtml.add(hourObj.get("code").getAsString());
+            }
+            JsonObject dayObj = backpageManager.getConnection("ajax/auction.php", Method.POST)
+                    .setParam("command", "getAuctionList")
+                    .setParam("category", "day")
+                    .setParam("list", "auction")
+                    .fromJson(JsonObject.class);
+            if (dayObj.get("result").getAsString().equalsIgnoreCase("ok")) {
+                dataHtml.add(dayObj.get("code").getAsString());
+            }
             lastAuctionUpdate = System.currentTimeMillis();
-            return data.parse(page);
+            return data.parse(dataHtml.toString());
```

Comment:
This just looks like a terrible hack, you're parsing a JSON output to produce a fake html output to re-parse again?

Consider refactoring this and just properly map the json data to the results you want, don't parse, then re-string it to parse it again